### DISCUSSION TITLE
Disney Junior Canada

### DISCUSSION THREAD

**Discussion by nickjrukfan** [iptv-org/community#714] [Q&A]
**Channel Name:** Disney Junior Canada
**Channel ID:** DisneyJunior.ca
**Country:** Canada
**Language:** English
**Website:** corus.ca

**Comment by freearhey** [comment]
@MapGuy11 once again, the label indicates the file to which the channel will need to be added, not the country of broadcast.

**Comment by MapGuy11** [comment]
> @MapGuy11 once again, the label indicates the file to which the channel will need to be added, not the country of broadcast.

Ok I am sorry about that. :)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/ca.m3u#L104-L105
